### PR TITLE
Issue #105 - adjust unit tests of tutorials

### LINKED ISSUE #105: Fix assertions in unit tests for tutorials
We removed some printing to stdout. This means assertions that checked that will no longer work.

Checklist:

- [x] pytorch tut1
- [x] pytorch tut2
- [x] pytorch tut3
- [x] TF2 keras
- [x] pytorch Mnist
- [x] TF2 Mnist
- [x] TF1 pipelining
- [x] TF2 tensor inspection
- [x] TF2 pipelining
- [x] pytorch transformers electra

### PR DESCRIPTION
**Pull request by Adamage**
Changes due to:
- removed ArgParser
- removed print statements

### CODE REVIEW

**Review by plutasnyy — APPROVED** [review]

**Comment by Adamage** [comment]
Situation changed:

- [x] wait until this is merged: https://github.com/Adamage/tutorials/pull/108
- [x] instead of removing the cases, keep them but add the Arg Parser, only make sure it will be hidden in Jupyter and Markdown

File: tutorials/pytorch/tut1_basics/tests/test_tuto_basics.py [hunk 1]
```diff
@@ -16,4 +16,4 @@ def test_model(self):
         # Check whether the model compiles and executes.
         self.run_command("python3 walkthrough.py",
                          working_path,
-                         "Model training and evaluation ran successfully")
```
Comment (Adamage) [inline tutorials/pytorch/tut1_basics/tests/test_tuto_basics.py:19]: Needs to be removed, it was not printed even before we changed this tutorial.

**Review by dirkbrink — APPROVED** [review]
Assuming these are the only 2 tutorials you've touched that had failing tests, then looks good to me

**Comment by Adamage** [comment]
@dirkbrink  yes. Other ones were in tutorial 8 (tests that started to fail due to arg parser removal). Now it's back everywhere (argparser)

**Pull request merged**